applications/RomApplication/python_scripts/rom_analysis.py

```diff
@@ -156,10 +163,14 @@ def ModifyAfterSolverInitialize(self):
 
             # Check for HROM stages
             if self.train_hrom:
+                # Pass the name of the Rom Parameters file
+                self.rom_parameters.AddString("rom_basis_output_name", self.rom_basis_output_name)
                 # Create the training utility to calculate the HROM weights
                 self.__hrom_training_utility = HRomTrainingUtility(
                     self._GetSolver(),
                     self.rom_parameters)
+                # Remove the name of the Rom Parameters file
+                self.rom_parameters.RemoveValue("rom_basis_output_name")
```

Comment:
Why adding and removing?

Reply:
It is not used further, I want to be as extrusive as possible.